### PR TITLE
MG1933053 陶光品

### PR DESCRIPTION
**Pull request by guangpintao**
ArrayList,102 same
PrintStream,134 same (OutputStream$write map,String$write nomap)
Calendar,50 same
File,142 same same
String,50 same same
PrintStream,6 same (no map)

### CODE REVIEW

**Comment by starry-xin** [comment]
MG1933054 田鑫 评估已通过

File: MG1933053/java/Calendar_set_int_int_int_int_int.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        assert(class0.get(Calendar.YEAR)==2019);
+        assert(class0.get(Calendar.MONTH)==11);
+        assert(class0.get(Calendar.DATE)==11);
+        assert(class0.get(Calendar.HOUR_OF_DAY)==19);
+        assert(class0.get(Calendar.MINUTE)==49);
+        System.out.println(class0.getTime());
+    }
+    /**
+     * input: 6
+     * class0: Calendar$class0
```
Comment (idiomaticrefactoring) [inline MG1933053/java/Calendar_set_int_int_int_int_int.java:44]: 去掉：

File: MG1933053/java/PrintStream_print_Array.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+ * @Map: OutputStream$func write(_ buffer: UnsafePointer<UInt8>, maxLength len: Int) -> Int
+ */
+public class PrintStream_print_Array {
+     /**
+     * input: 2
+     * class0  PrintStream$class0
+     * char [] s=['a','b','\n']
+     * output: 0
+     * PrintStream$class0-PrintStream$PrintStream(OutputStream out) &OutputStream out=System.out & https://docs.oracle.com/javase/8/docs/api/java/io/PrintStream.html#PrintStream-java.io.OutputStream-
+     */
```
Comment (idiomaticrefactoring) [inline MG1933053/java/PrintStream_print_Array.java:21]: OutputStream$out 或者直接out

**Review by idiomaticrefactoring — CHANGES_REQUESTED** [review]
perfect-2

**Pull request merged**